ReadMe.md

```diff
@@ -65,44 +65,47 @@ DESCRIPTION:
    Run as 'root' or 'clickhouse' user
 
 COMMANDS:
-   tables               List list of tables, exclude skip_tables
+   tables               List of tables, exclude skip_tables
```

Comment:
This part of README is generated
from `cmd/clickhouse-backup/main.go`
please change it instead

Reply:
Oh I see how this works. I've regenerated file, moved contents to just Manual.md (rather than keep it in Readme and Manual). I've left a link in the readme file to point to the manual.